src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
I don't think I see any of these as good reasons to make the change. Responding to your points below...

1. On your first point, I would argue that this change *makes* the experience inconsistent and doesn't add information that actually helps the user. The high-order bit for the user is that an API is "deprecated", not that it was decorated with the ObsoleteAttribute or DeprecatedAttribute.
2. The fact that there are two attributes is a reality. That's just the way it is and it's not going to change.
3. "The design is not correct" is subjective and doesn't describe why this is valuable for users. This really isn't a great argument and I provided a link to code that would make Quick info *inches* taller on screen if we started showing attributes.

Finally, if taking this change implicitly gives the impression that we are changing the design of Quick Info to show all attributes, I would prefer not to take the change.